WreckEngine/Text.cpp

```diff
@@ -548,19 +541,39 @@ float Text::getKerning(uint32_t cp1, uint32_t cp2, const FontFace* font)
     return kerning;
 }
 
+void Text::flush() {
+    instances.flush();
+}
+
+void Text::postUpdate() {
+    instances.seal();
+}
+
+void Text::render(Render::MaterialPass* matRenderer)
+{
+    if (Text::active)
+    {
+        renderer.renderer = matRenderer;
+        renderer.draw();
+    }
+    else {
+        instances.consumeAll([](auto&) {}); // clears the list without doing anything
```

Comment:
It doesn't matter too much now because we aren't rendering too many text instances, but wouldn't it be more performant to just have a clear method instead of iterating everything with an empty function?